src/domain/event/utils.ts

```diff
@@ -55,3 +56,24 @@ export const getEventStartTimeStr = (
  */
 export const isEventFree = (event: EventFieldsFragment): boolean =>
   Boolean(event.offers.find((item) => item.isFree)?.isFree);
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export const getEventFields = (
+  event: EventFieldsFragment | undefined | null,
+  locale: Language
+) =>
+  event
+    ? {
+        isEventFree: isEventFree(event),
+        eventName: getLocalisedString(event?.name, locale),
+        shortDescription: getLocalisedString(event?.shortDescription, locale),
+        description: getLocalisedString(event?.description, locale),
+        imageUrl: event?.images?.[0]?.url,
```

Comment:
Maybe could use placeholder image as default in case that image doesn't exist. You can get placeholder image by using getEventPlaceholderImage util functton